.env

```diff
@@ -0,0 +1,3 @@
+CONTENTFULLY_SPACE_ID=9fk7ppaemaro
+CONTENTFUL_ACCESS_TOKEN=6lspbxD-_P0NxKg6PsL4TM34DFRGu_Co5D2RtgZfPnE
```

Comment:
this is a big mistake: never ever push secret keys to github
This file (.env) should be ignored in (.gitignore)